src/money.js

```diff
@@ -0,0 +1,110 @@
+"use strict";
```

Comment:
Would be nice to use something like rollup(https://github.com/rollup/) so we can provide commonjs and/or es6 modules as required

Reply:
Happy to, My experience with using webpack and it like is that you still compile to down to es5 with babel, But most of the time the transforms exclude node_modules so the module would only load in the es2015 version which may break some apps depending on where you use it.

Reply:
That said - since you guys are the primary users of this lib - happy to either way.